.circleci/config.yml

```diff
@@ -1,66 +1,122 @@
 version: 2.1
+
+orbs:
+  aws-cli: circleci/aws-cli@0.1.22
+  k8s: circleci/kubernetes@0.11.0
+
+commands:
+  checkout_and_cache_dependencies:
+    description: "checkout code and cache go dependencies"
+    steps:
+      - checkout
+      - restore_cache:
+          key: go-mod-{{ checksum "go.sum" }}
+      - run:
+          name: go mod download
+          command: go mod download
+      - save_cache:
+          key: go-mod-{{ checksum "go.sum" }}
```

Comment:
nice work above with the go mod caching. :+1:

Reply:
😄